README.md

```diff
@@ -43,38 +75,48 @@ The only compile time dependency you will need is `okta-authn-sdk-api`.  You wil
 </dependency>
 ```
 
-## SNAPSHOT Dependencies
+For Gradle:
 
-Snapshots are deployed off of the 'master' branch to OSSRH and can be consumed using the following repository configured for Apache Maven or Gradle:
-```
-https://oss.sonatype.org/content/repositories/snapshots/
+```groovy
+compile 'com.okta.sdk:okta-authn-sdk-api:${okta.version}'
+runtime 'com.okta.sdk:okta-authn-sdk-impl:${okta.version}'
+runtime 'com.okta.sdk:okta-sdk-httpclient:${okta.version}'
 ```
 
-## Client configuration
+### SNAPSHOT Dependencies
 
-There are a few ways to configure the client, but the easiest way is to create a `~/.okta/okta.yaml` file and `orgUrl` value:
+Snapshots are deployed off of the 'master' branch to [OSSRH](https://oss.sonatype.org/) and can be consumed using the following repository configured for Apache Maven or Gradle:
 
-``` yaml
-okta:
-  client:
-    orgUrl: https://dev-123456.oktapreview.com
+```txt
+https://oss.sonatype.org/content/repositories/snapshots/
 ```
 
-## Creating a Client
-
- Once you create your `okta.yaml` file, you can create a client with a couple of lines:
+You'll also need:
 
-``` java
-// Instantiate a builder for your client. If needed, settings like Proxy and connection timeouts can be defined here.
-AuthenticationClientBuilder builder = AuthenticationClients.builder();
+* An Okta account, called an _organization_ (sign up for a free [developer organization](https://developer.okta.com/signup) if you need one)
 
-// No need to define anything else; build the AuthenticationClient instance.
-AuthenticationClient client = builder.build();
+ 
+Construct a client instance by passing it your Okta domain name and API token:
+ 
+[//]: # (NOTE: code snippets in this README are updated automatically via a Maven plugin by running: mvn okta-code-snippet:snip.okta:okta-code-snippet-maven-plugin:snip)
+ 
+[//]: # (method: createClient)
+```java
+AuthenticationClient client = AuthenticationClients.builder()
+    .setOrgUrl("https://dev-123456.oktapreview.com/")
+    .build();
 ```
+[//]: # (end: createClient)
+ 
+Hard-coding the Okta domain works for quick tests, but for real projects you should use a more secure way of storing these values (such as environment variables). This library supports a few different configuration sources, covered in the [configuration reference](#configuration-reference) section.
+ 
+## Usage guide
 
-For more details see: [Creating a Client](https://github.com/okta/okta-auth-java/wiki/Creating-a-Client)
+These examples will help you understand how to use this library. You can also browse the full [API reference documentation][javadocs].
 
-## Authentication
+Once you initialize a `AuthenticationClient`, you can call methods to make requests to the Okta API.
```

Comment:
suggestion: to the Okta **Authentication** API. To call other Okta APIs, see the [Management SDK].